evm.json

```diff
@@ -1006,5 +897,66 @@
       "chainId": 99,
       "networkId": 99
     }
+  },
+  {
+    "name": "Ethereum Mainnet",
+    "network": "eip155-1",
+    "interface": "evm",
+    "coins": [
+      {
+        "name": "Ether",
+        "symbol": "ETH",
+        "denom": "wei",
+        "exponent": 18
+      }
+    ],
+    "rpc": [
+      "https://mainnet.infura.io/v3/${INFURA_API_KEY}",
+      "https://api.mycryptoapi.com/eth"
+    ],
+    "custom": {
+      "chainId": 1,
+      "networkId": 1
+    }
+  },
+  {
+    "name": "Ethereum Classic Mainnet",
+    "network": "eip155-61",
+    "interface": "evm",
+    "coins": [
+      {
+        "name": "Ethereum Classic Ether",
+        "symbol": "ETC",
+        "denom": "wei",
+        "exponent": 18
+      }
+    ],
+    "rpc": [
+      "https://ethereumclassic.network"
+    ],
+    "custom": {
+      "chainId": 61,
+      "networkId": 1
+    }
+  },
+  {
+    "name": "Ubiq Network Mainnet",
+    "network": "eip155-8",
+    "interface": "evm",
+    "coins": [
+      {
+        "name": "Ubiq Ether",
+        "symbol": "UBQ",
+        "denom": "wei",
+        "exponent": 18
+      }
+    ],
+    "rpc": [
+      "https://pyrus2.ubiqscan.io"
+    ],
+    "custom": {
+      "chainId": 8,
+      "networkId": 1
+    }
```

Comment:
Why such a big diff? For some reason the order of the chains changed.

Reply:
Arghh I didn't notice that, but on `scripts/shared/index.js` you can see on L184 that `saveList` orders the output json by `id` which was removed. I will change it order by `network`

Reply:
Plus I forgot to update the standard spec

Reply:
Cool, ordering by `network` reduced the diff

Reply:
My IDE also auto-formatted the spec md 😅 I just reverted it